### DISCUSSION TITLE
Feature Request: Jellyfin Plugin

### DISCUSSION THREAD

**Discussion by Snuupy** [Zibbp/ganymede#66] [Ideas]
I consume most of my media in Jellyfin because it's my "home" for all my content. It has a great video player, offers video transcoding during playback, supports a bunch of different clients, and tracks watched history/status, similar to Plex/Emby, and is FOSS.

Would a Jellyfin plugin be beyond the scope of this project? I'm thinking just that the chat would need to be displayed on the right side during video playback and Jellyfin would be able to handle the rest of the video playback.

What do you guys think?

**Comment by Zibbp** [comment]
Hi,
It is a cool idea but it is out of scope for this project. I also do not know any c# which is what Jellyfin plugins are written in so I wouldn't be able to even start.

The video and chat render can be played back via Jellyfin but getting them to sync and be next to each other is another issue.
